recipes/qwen/Qwen2.5-1.5B-Instruct/grpo/confg_full.yaml

```
@@ -0,0 +1,45 @@
+# Model arguments
+model_name_or_path: <your-hub-id>/Qwen2.5-1.5B-Open-R1-Distill
```

nit to ensure the example in the README runs:
```suggestion
model_name_or_path: deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B
```

---

`DeepSeek-R1-Distill-Qwen-1.5B` uses `Qwen2.5-Math-1.5B` as the base model, while the current sft uses `Qwen2.5-1.5B-Instruct`. I'm not sure if that leads to some bias.